### DISCUSSION TITLE
Add UI feature - set alert threshold on a dashboard (was available in grafana 7)

### DISCUSSION THREAD

**Discussion by simonsk1990** [grafana/grafana#67968] [Dashboards, Panels & Visualizations]
In Grafana 7 there was a nice UI feature (when it was available setting alerts via the dashboards) - When have a graph - if alert was set to see its threshold on the graph - see example here:
![image](https://user-images.githubusercontent.com/58727993/236759006-8765b9d2-5506-4891-990e-0b59f60e8fc1.png)

Since moving to the new "multi-dimensional" alerting system (and I have my own opinions for this) - the decoupling of the alerts and the dashboards, this feature is no longer supported:
![image](https://user-images.githubusercontent.com/58727993/236759268-8268514d-2107-439a-8335-07c4d55ba2be.png)

It will be nice to have it back **if possible** (not sure there is). There is however a workaround to add a threshold manually but its not consistent with the alert.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82684. Feel free to continue the discussion around this there. Thank you!
